src/App.js

```diff
@@ -1,29 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+// import './App.css';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.